python/README.md

```diff
@@ -0,0 +1,107 @@
+## Prelude
+Invoca Python Style Guide
+
+The contents of this styleguide are to be enforced by [flake8](https://flake8.pycqa.org/en/latest/) and [black](https://github.com/psf/black) through the use of the flake8 plugin [flake8-black](https://github.com/peterjc/flake8-black).
+
+Any adjustments to this styleguide should be captured in the [tox.ini](tox.ini) config.
+
+## Table of Contents
+
+- [Prelude](#prelude)
+- [Table of Contents](#table-of-contents)
+- [PEP-8: Style Guide for Python Code](#pep-8-style-guide-for-python-code)
+- [PEP-20: The Zen of Python](#pep-20-the-zen-of-python)
+- [Layout](#layout)
+- [Syntax](#syntax)
+- [Naming](#naming)
+- [Comments](#comments)
+- [Classes](#classes)
+- [Type Annotations](#type-annotations)
+- [Recommendations](#recommendations)
+
+## PEP-8: Style Guide for Python Code
+
+[PEP-8](https://peps.python.org/pep-0008/) defines the official style guide for python. While it is not entirely comprehensive, it provides a sufficient foundation for more detailed style guides such as this. This style guide makes frequent reference to PEP-8. In general code should adhere to that. Any extensions to or deviations from PEP-8 will be documented here.
+
+## PEP-20: The Zen of Python
+
+[PEP-20](https://peps.python.org/pep-0020/#the-zen-of-python) defines the so-called Zen of Python which is a set of principles for writing clean, idiomatic python code. Where PEP-8 is prescriptive, this is much more abstract and interpretive. While this shouldn't be taken as scripture, it useful to apply these abstract principles when faced with ambiguity.
```

Comment:
```suggestion
[PEP-20](https://peps.python.org/pep-0020/#the-zen-of-python) defines the so-called Zen of Python which is a set of principles for writing clean, idiomatic python code. Where PEP-8 is prescriptive, this is much more abstract and interpretive. While this shouldn't be taken as scripture, it may be useful to apply these abstract principles when faced with ambiguity.
```